.github/CONTRIBUTING.md

```diff
@@ -90,6 +90,16 @@ Updating the Katas to a different QDK version can be done using PowerShell scrip
 
 After running this script you should validate that the update didn't introduce any breaking changes; see the next section for how to do this.
 
+### Excluding the tasks so that CI doesn't fail
+Currently some tasks are excluded from validation by adding the appropriate cell tag to the task due to several reasons : 
+- Some tasks require implementing two code cells at once before running the test, so the first of the cells implemented is guaranteed to fail(```two_player_game```)
```

Comment:
I would suggest the following list of tags, with (hopefully) more accurate tags:
* `timeout` and `invalid_code` are good
* `multicell_solution` instead of `two_player_game` (I can envision three-player-games or tasks that are not games, and technically the issue with them is that there are multiple cells that need to be executed at once). The later tasks in CHSHGame and MagicSquareGame would also be a special case of this (because they rely on earlier tasks in the notebook which we exclude from running)
* `randomized_solution` instead of `failure_of_correct_solution` (correct solution shouldn't fail, by definition,  but correct randomized solution is expected to fail and it's less scary)
* I think we don't have any `work_in_progress` right now? If we had, this seems like it would be `invalid_code`, so we could  get away with just 4 tags

Reply:
Replaced `two_player_game` with `multicell_solution` and `failure_of_correct_solution` with `randomized_solution`. Also, removed the `work_in_progress` tag.